tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -14,14 +14,19 @@ function createDefaultUser(userId) {
   const password = userSettings.getPasswordForUser(userId)
   const displayname = userSettings.getDisplayNameOfDefaultUser(userId)
   const email = userSettings.getEmailAddressOfDefaultUser(userId)
-  if (client.globals.ocis) {
+  if (client.globals.ldap) {
     return ldap.createUser(client.globals.ldapClient, userId)
   }
   return createUser(userId, password, displayname, email)
 }
 
 function createUser(userId, password, displayName = false, email = false) {
   const body = new URLSearchParams()
+  if (client.globals.ocis) {
+    email = userId + '@example.com'
```

Comment:
this should only happen if `email === false`

Reply:
fixed